azext_edge/edge/_help.py

```diff
@@ -650,6 +650,67 @@ def load_iotops_help():
         short-summary: Dataflow endpoint management.
     """
 
+    helps[
+        "iot ops dataflow endpoint apply"
+    ] = """
+        type: command
+        short-summary: Create or replace a dataflow endpoint resource.
+        long-summary: |
+          An example of the config file format is as follows:
+
+          {
```

Comment:
just include this as a file in docs (maybe make a sub folder called examples)

Reply:
this is now align with broker commands, for TODO this will be move to a aka.ms link to a markdown file

Reply:
I've also followed up with the Az CLI team to understand why the format gets mangled in reference docs, when it is nicely formatted in the CLI. TBD.

Reply:
@Elsie4ever let's try surrounding the json with markdown backticks ```

Reply:
@digimaun this is how -h looks after adding backtick
<img width="500" alt="image" src="https://github.com/user-attachments/assets/e0a2c0e8-761f-4245-852f-d7f3cdb16e9e" />